Patches/Moyo light in the abyss/Scenario/Moyo_Scenario.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Moyo-Light in the Abyss</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ScenarioDef[defName="Moyo_RefugeeScenario"]/scenario/parts</xpath>

```

Comment:
Moyo - Light in the Abyss doesn't appear to have its own scenario in it--as far as I can tell, this 'Moyo_RefugeeScenario' comes with the base Moyo - From the Depths mod, which is a pre-req for Abyss and is already patched.

Meaning, this entire file can be deleted, since the scenario is already has ammo added by the base Moyo patch, and this file creates duplicates.

Reply:
Yeah I'll delete this one. Rest of the file applies as well.